module/VuFind/src/VuFind/Role/PermissionProvider/SecureCookie.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Secure cookie permission provider for VuFind.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Catalyst IT 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Authorization
+ * @author   Alex Buckley <alexbuckley@catalyst.net.nz>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://www.vufind.org  Main Page
+ */
+
+namespace VuFind\Role\PermissionProvider;
+
+use VuFind\Cookie\CookieManager;
+
+/**
+ * Secure cookie permission provider for VuFind.
```

Comment:
Ditto.

Reply:
thanks @demiankatz that is removed in my follow-up